source-git-monorepos/README.md

```diff
@@ -50,21 +89,355 @@ packages:
 
 With the above source-git configuration could be kept in
 `.distro/source-git.yaml`. Downstream files to be synced to dist-git are to be
-kept in subdirectories, named according to the distro package, in `.distro` to
+kept in subdirectories, named according to `package_name`, in `.distro`, to
 keep things organized.
 
 This way all distribution configs and files would be kept at one place, which
-probably is somewhat better than having source-git configuration spread
-across the monorepo, in subdirectories. This setup also allows multiple paths
-from the repo mapped to a dist-git repo.
+probably is somewhat better than having source-git configuration spread across
+the monorepo, in subdirectories. This setup also allows multiple paths from
+the repo mapped to a dist-git repo.
```

Comment:
I'm confused here: so where would I find a specfile?

Reply:
By convention in source-git specfiles would be stored in `.distro/<package_name>/<dist_git_name>.spec`. I've added an example above to make this more clear.

In a Packit context, it doesn't really matter, as everything is configurable.

Reply:
That makes sense, I like that the spec files are in a single place. I thought about how archives for these packages are created - but Packit can already work with setups where spec files and sources are in different directories.